vertx-mssql-client/src/main/java/io/vertx/mssqlclient/MSSQLException.java

```diff
@@ -29,7 +31,7 @@ public class MSSQLException extends RuntimeException {
   private List<MSSQLException> additional;
 
   public MSSQLException(int number, byte state, byte severity, String message, String serverName, String procedureName, int lineNumber) {
-    super(null, null, false, false);
+    super(null, number, null);
```

Comment:
we should format an error message like we do for `PgException ` and `DB2Exception`